### PR TITLE
Updating on supported restart policy

### PR DESCRIPTION
**Pull request by IceCodeNew**

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks @IceCodeNew 
You need to sign your PRs and squash them

git rebase -i origin
to squash
git commit -a --amend -s
git push --force

File: cmd/podman/common/create.go [hunk 1]
```diff
@@ -389,7 +389,7 @@ func GetCreateFlags(cf *ContainerCLIOpts) *pflag.FlagSet {
 	createFlags.StringVar(
 		&cf.Restart,
 		"restart", "",
-		`Restart policy to apply when a container exits ("always"|"no"|"on-failure")`,
+		`Restart policy to apply when a container exits ("always"|"no"|"on-failure"|"unless-stopped"|"on-success"|"on-abnormal"|"on-watchdog"|"on-abort")`,
```
Comment (TomSweeneyRedHat) [inline cmd/podman/common/create.go:392]: alphabetize please:
```suggestion
		`Restart policy to apply when a container exits ("always"|"no"|"on-abnormal"|"on-abort"|"on-failure"|"on-success"|"on-watchdog"|"unless-stopped")`,
```

File: docs/source/markdown/podman-generate-systemd.1.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Override the default stop timeout for the container with the given value.
 **--restart-policy**=*policy*
 
 Set the systemd restart policy.  The restart-policy must be one of: "no", "on-success", "on-failure", "on-abnormal",
-"on-watchdog", "on-abort", or "always".  The default policy is *on-failure*.
+"on-watchdog", "on-abort", "unless-stopped", or "always".  The default policy is *on-failure*.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-generate-systemd.1.md:42]: alphabetize these too like prior comment.

**Comment by TomSweeneyRedHat** [comment]
and most importantly, TYVM @IceCodeNew for the PR!

File: docs/source/markdown/podman-generate-systemd.1.md (hunk 2 above)
Comment (Luap99) [reply docs/source/markdown/podman-generate-systemd.1.md:42]: `podman generate systemd --restart-policy` does not support `unless-stopped`. Systemd only supports the policies as currently documented. see: `man systemd.service`

File: cmd/podman/common/create.go (hunk 1 above)
Comment (Luap99) [reply cmd/podman/common/create.go:392]: `podman create/run --restart` only supports `always`, `no`, `on-failure` and `unless-stopped`. At least according to our documentation (`man podman-create`)

**Review by Luap99 — COMMENTED** [review]
@IceCodeNew It looks like you mixed up `podman generate systemd --restart-policy` and `podman create/run --restart`.

File: docs/source/markdown/podman-generate-systemd.1.md (hunk 2 above)
Comment (IceCodeNew) [reply docs/source/markdown/podman-generate-systemd.1.md:42]: > `podman generate systemd --restart-policy` does not support `unless-stopped`. Systemd only supports the policies as currently documented. see: `man systemd.service`

Oops, I found that out yesterday too. (https://github.com/containers/podman/pull/6522#issuecomment-699190770)
I'm wondering if this is something that PR #6522 doesn't implement and we should refine our support for this policy in the future?

**Comment by Luap99** [comment]
LGTM
@rhatdan @TomSweeneyRedHat PTAL

**Pull request merged**